docs/planning/HANDOFF_UNIFY_DOCUMENT_CONTEXT_PRESERVATION.md

```
@@ -0,0 +1,734 @@
+        }
+      }
+    }
+  }
+}
+```
+
+---
+
+## 4. Example Output
+
+### Input Document
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)